homeassistant/components/zwave/__init__.py

```diff
@@ -182,10 +182,9 @@ def nice_print_node(node):
     node_dict['values'] = {value_id: _obj_to_dict(value)
                            for value_id, value in node.values.items()}
 
-    print("\n\n\n")
-    print("FOUND NODE", node.product_name)
-    pprint(node_dict)
-    print("\n\n\n")
+    _LOGGER.info("FOUND NODE %s \n"
+                 "%s \n"
+                 "\n", node.product_name, node_dict)
```

Comment:
@andrey-git The logs don't get truncated, because we don't set `%(message).256s` (this would truncate the message to 256 chars): 

https://github.com/home-assistant/home-assistant/blob/27c18068971096219b720e0f18ebb68cde34739e/homeassistant/bootstrap.py#L226-L227

-------------
I do however not like new lines in logs :/ they just end up looking terrible and difficult to parse. But I don't know of another way we can print out all this (probably quite useful) information while still not spamming the logs too much...

Reply:
I had too many newlines in the output. Removed some of them.